extensions/resteasy-reactive/rest-client-reactive/runtime/src/main/java/io/quarkus/rest/client/reactive/runtime/RestClientBuilderImpl.java

```diff
@@ -360,6 +360,13 @@ public <T> T build(Class<T> aClass) throws IllegalStateException, RestClientDefi
             clientBuilder.setUserAgent(restClientsConfig.userAgent.get());
         }
 
+        Integer maxChunkSize = (Integer) getConfiguration().getProperty(QuarkusRestClientProperties.MAX_CHUNK_SIZE);
+        if (maxChunkSize != null) {
+            clientBuilder.maxChunkSize(maxChunkSize);
+        } else {
+            clientBuilder.maxChunkSize(8096);
```

Comment:
Probably best have the number n a constant as it used multiple times in the source

Reply:
Done.